src/main/java/net/sigmarik/abilitymod/AbilityMod.java

```diff
@@ -33,10 +37,78 @@ public class AbilityMod implements DedicatedServerModInitializer {
 
 	public static final double BOAT_ATTRACTION_FACTOR = 0.1;
 
+	public static final int DAYLIGHT_BURNING_CHANCE_PER_TICK = 55;
+
 	public static final int ADDICTION_START_TIMER 	= 1800 * 20;	// 30 minutes
 	public static final int ADDICTION_MID_TIMER 	= 180 * 20;   	// 3 minutes
 	public static final int ADDICTION_WARNING_TIMER = 20 * 20;
 
+	public static final HashSet<Item> IRON_ITEMS = new HashSet<>(List.of(
+			Items.ANVIL,
+			Items.AXOLOTL_BUCKET,
+			Items.BLAST_FURNACE,
+			Items.BUCKET,
+			Items.CAULDRON,
+			Items.CHAIN,
+			Items.CHAINMAIL_BOOTS,
+			Items.CHAINMAIL_CHESTPLATE,
+			Items.CHAINMAIL_HELMET,
+			Items.CHAINMAIL_LEGGINGS,
+			Items.CHEST_MINECART,
+			Items.CHIPPED_ANVIL,
+			Items.COMPASS,
+			Items.CROSSBOW,
+			Items.DAMAGED_ANVIL,
+			Items.DEEPSLATE_IRON_ORE,
+			Items.DETECTOR_RAIL,
+			Items.FLINT_AND_STEEL,
+			Items.FURNACE_MINECART,
+			Items.HEAVY_WEIGHTED_PRESSURE_PLATE,
+			Items.HOPPER,
+			Items.HOPPER_MINECART,
+			Items.IRON_AXE,
+			Items.IRON_BARS,
+			Items.IRON_BLOCK,
+			Items.IRON_BOOTS,
+			Items.IRON_CHESTPLATE,
+			Items.IRON_DOOR,
+			Items.IRON_HELMET,
+			Items.IRON_HOE,
+			Items.IRON_HORSE_ARMOR,
+			Items.IRON_INGOT,
+			Items.IRON_LEGGINGS,
+			Items.IRON_NUGGET,
+			Items.IRON_ORE,
+			Items.IRON_PICKAXE,
+			Items.IRON_SHOVEL,
+			Items.IRON_SWORD,
+			Items.IRON_TRAPDOOR,
+			Items.LANTERN,
+			Items.LAVA_BUCKET,
+			Items.MILK_BUCKET,
+			Items.MINECART,
+			Items.PISTON,
+			Items.POWDER_SNOW_BUCKET,
+			Items.RAIL,
+			Items.RAW_IRON,
+			Items.RAW_IRON_BLOCK,
+			Items.SHEARS,
+			Items.SHIELD,
+			Items.SMITHING_TABLE,
+			Items.SOUL_LANTERN,
+			Items.STICKY_PISTON,
+			Items.STONECUTTER,
+			Items.TNT_MINECART,
+			Items.TRAPPED_CHEST,
+			Items.TRIPWIRE_HOOK,
+			Items.WATER_BUCKET
+	));
+
+	public static final double MOVEMENT_SPEED_BASE_MULTIPLY_COEFFICIENT = 0.3;
```

Comment:
`THIS_IS_HELL_OF_A_LONG_VARIABLE_NAME`
And even the name above is shorter than the name of the variable in the code.

Reply:
He doesn't like obscure names, he doesn't like understandable names...

Reply:
The very reason why I asked you to change the variable name was that it didn't represent the trait it relates to.
Now let's take a look at you 40-character long (holy ****) variable name. May be I am a dummy, after all, and you really need that many words to describe what is going on.
How does the word "BASE" contribute to the variable name? Is this variable the root of some obscure tree? Or is it some value that will be gradually changed later? But why is it `final`, then?
That's because it is not, it's just a constant you need to multiply the player's movement at one point in the code. So the word "BASE" is unnecessary at best and misleading at worst.
"MULTIPLY_COEFFICIENT". It's called "multiplication coefficient" if you really want to preserve that Shakespearian vibe you are going with in your code. Programmers usually call this "multiplier", unless it occurs in some form of series or equations with other parameters, where it should be called "coefficient". In this case the word "multiplier" is more than enough.
"MOVEMENT_SPEED". There are not many parameters related to "movement". At least not many you would ever want to change. Just the word "movement" or "speed" should be enough.

So, we end up with "SPEED_MULTIPLIER" or "MOVEMENT_SPEED_MULTIPLIER", if you don't like the last transformation. These names are shorter than the original and less misleading than the original. Yet, neither them, nor the original 40-ch. long name describe where this multiplier is applied. It could relate to cobweb speed reduction (I mean, this coefficient is less than 1, so it should relate to slowing down something), it could relate to speed potion effect change, it could just mean that every player is now 3 times slower than they originally were. This extremely long misleading nonsensical variable name does not do ANYTHING to describe the very thing I asked for in the original review. So what's the purpose of all this circus?